.github/workflows/pylint.yml

```diff
@@ -0,0 +1,17 @@
+name: Pylint
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip && pip install pylint
+    - name: Analysing the code with pylint
+      run: pylint $(git ls-files '*.py')
```

Comment:
How about checking the modified files instead of all files?

Reply:
good point !! i will fix